tests/PySys/software_management_end_to_end/environment_sm_management.py

```diff
@@ -407,6 +407,13 @@ def check_is_installed(self, package_name, version=None):
                 break
         return ret
 
+    def remove_package_apt(name):
+        """Remove a package with apt
+        """
+        assert isinstance(name, str)
+        proc = subprocess.run(["/usr/bin/sudo", "apt", "-y", "remove", name])
```

Comment:
can this '/usr/bin/sudo' be declared upfront and reused here?

Reply:
Yes we could. However, I believe this is outside of the scope of this PR. There are many more things that we should refactor.